### ISSUE TITLE
Initialization problem on 5.0.2 on iOS

### ISSUE THREAD

**Issue by rafaelcmrj** [EddyVerbruggen/nativescript-plugin-firebase#563]
Once I updated to 5.0.2 version I started getting the following error

`Error in firebase.init: ReferenceError: Can't find variable: FIRFirestoreSettings`

Taking a look at the source code, it seems to be coming from firebase.ios.js

`if (typeof (FIRStorage) !== "undefined") {
                if (arg.persist === false) {
                    var fIRFirestoreSettings = FIRFirestoreSettings.new();
                    fIRFirestoreSettings.persistenceEnabled = false;
                    FIRFirestore.firestore().settings = fIRFirestoreSettings;
                }
            }`

My init method is the following:

`firebase.init({
		persist: false,
		iOSEmulatorFlush: true,
		storageBucket: '******url-here*****',
		onAuthStateChanged: function (data) {
			if (data.loggedIn && !data.user.anonymous) {
				tmutils.setSession(data.user.uid, function () {
					console.log('*** user all set, redirect to main view')
					tmutils.navigateTo({
						moduleName: Routes.MainView,
						clearHistory: platform.isIOS
					});
				});
			};
		}
	}).then(instance => {
		console.log('**** THEN FIREBASE INIT ****');
		application.notify({
			eventName: 'doLogin'
		});
	});`

**Comment by RaphaelJenni** [comment]
Same on Android, but my problem is with the Messages. 

https://github.com/EddyVerbruggen/nativescript-plugin-firebase/commit/6caf824bf1b8d6fcc566dd27896977b626f5d971#diff-f2ded97e613fc456df16ddd58ad83b52R30

It can't find the firebase messages library as I haven't added it to the list of used firebase services.

Need to go back to version 5.0.1

**Comment by EddyVerbruggen** [comment]
@RaphaelJenni Can you please elaborate? I'm not sure what you mean.

**Comment by EddyVerbruggen** [comment]
@rafaelcmrj I screwed up: that code should not run on your device because you didn't enable Firestore. I had guarded that code with

```typescript
if (typeof(FIRStorage) !== "undefined") {
```

, but it should be

```typescript
if (typeof(FIRFirestore) !== "undefined") {
```

Fixed in 5.0.3 which I'll publish shortly.
